ansible/roles/nginx/defaults/main.yml

```diff
@@ -53,6 +53,31 @@ nginx_host_allow: []
 #
 nginx__deploy_state: 'present'
 
+                                                                   # ]]]
+# .. envvar:: nginx__packages_installation [[[
+#
+# There is a deployment scenario possible where you install nginx packages from
+# sources or manually. This option allows you to omit nginx packages
+# installation by the role. Use with caution, as it can have unexpected
+# consequences. Notice: you must define ``nginx_default_tls_protocols``
+# manually, for example:
+#
+# ``nginx_default_tls_protocols: [ "TLSv1.2", "TLSv1.3" ]``
```

Comment:
Is that step actually needed? The fact script installed by the role should be able to get `nginx` version from `/usr/sbin/nginx`, is the binary from `nginx-ee` installed elsewhere? In that case perhaps we can rely on `$PATH` to find the correct `nginx` binary instead.

Reply:
The role gave errors when I tested it out when no nginx was installed at all. So maybe instead of **must** I could use wording **could**, indeed and add a phrase ```when the path is different than /usr/sbin/nginx```.